tox.ini

```diff
@@ -1,13 +1,14 @@
 [tox]
 envlist =
     {py27,py32,py33,py34,py35}-django18
-    {py27,py34,py35}-{django19,django110,django-latest}
+    {py27,py34,py35}-{django19,django110,django111,django-latest}
```

Comment:
`django-latest` only needs testing against `py35` I think. (2.0 will require this)